utils_for_test.go

```diff
@@ -69,6 +69,6 @@ func InboundConnection(call IncomingCall) (*Connection, net.Conn) {
 }
 
 // NewSpan returns a Span for testing.
-func NewSpan(traceID uint64, parentID uint64, spanID uint64) Span {
-	return Span{traceID: traceID, parentID: parentID, spanID: spanID, flags: defaultTracingFlags}
+func NewSpan(traceID uint64, spanID uint64, parentID uint64) Span {
```

Comment:
Don't have a strong opinion. In the Jaeger's http header the order is `trace:span:parent:flags`. In fact, many people argue that it is not even necessary to propagate `parent` across RPC, it was enough to record it out of band.